tests/cases/php/closing-tag

```diff
@@ -9,11 +9,10 @@ $foo = 'bar';
 
 ?>
 
----FIXED---
 <?php
```

Comment:
Strange that `?> ... <?php` isn't automatically fixed

Reply:
Could in this case as it's just whitespace, but probably a pretty-edge case!

Reply:
very true!